tests/components/opentherm_gw/test_config_flow.py

```diff
@@ -39,20 +39,21 @@ async def test_form_user(hass: HomeAssistant) -> None:
 
     with (
         patch(
-            "homeassistant.components.opentherm_gw.async_setup",
-            return_value=True,
-        ) as mock_setup,
+            "homeassistant.components.opentherm_gw.config_flow.pyotgw.OpenThermGateway",
+            return_value=MagicMock(
+                connect=AsyncMock(return_value=MINIMAL_STATUS),
+                disconnect=AsyncMock(),
+            ),
+        ) as gateway,
         patch(
-            "homeassistant.components.opentherm_gw.async_setup_entry",
-            return_value=True,
-        ) as mock_setup_entry,
-        patch(
-            "pyotgw.OpenThermGateway.connect", return_value=MINIMAL_STATUS
-        ) as mock_pyotgw_connect,
-        patch(
-            "pyotgw.OpenThermGateway.disconnect", return_value=None
-        ) as mock_pyotgw_disconnect,
-        patch("pyotgw.status.StatusManager._process_updates", return_value=None),
+            "homeassistant.components.opentherm_gw.OpenThermGateway",
+            return_value=MagicMock(
+                connect=AsyncMock(return_value=MINIMAL_STATUS),
+                set_control_setpoint=AsyncMock(),
+                set_max_relative_mod=AsyncMock(),
+                disconnect=AsyncMock(),
+            ),
+        ),
```

Comment:
I now also notice you patch the normal gateway, I think that's because the `async_setup_entry` is not mocked. Also check airgradient on how I do that :)